README.md

```diff
@@ -52,6 +52,29 @@ following snippet to your config:
 (vhdl-ext-lsp-set-server 've-rust-hdl)   ; `lsp' config
 ```
 
+## Use in Neovim
+
+Neovim provides an LSP client to the VHDL_LS language server. Install the VHDL_LS  
+binary e.g. via Cargo. The binary must be on the path and executable (if you  
+can run "vhdl_ls -h" in the terminal then you're good). Put the "vhdl_libraries"  
```

Comment:
I'm not quite sure everyone understands what The "vhdl_libraries" are. Could you add a small explanation here?

Reply:
Thanks for bringing that up, it is something I didn't quite fully understand. There is the [vhdl_libraries](https://github.com/VHDL-LS/rust_hdl/tree/master/vhdl_libraries)  directory in this repo. When I use VHDL_LS the way I described it expects this directory in the parent directory of the VHDL_LS binary, otherwise it will crash in Neovim. Simply copying it there worked for me, but I guess it is not exactly how it is supposed to be, right?

Reply:
Well, yes and no :D
The language server needs the standard libraries to correctly work. That way it can, for example, resolve `std_logic` or other symbols from the standard library when you use them. So therefore simply copying is fine. However, the better approach would be to package the libraries with the executable. This is done for the artefacts and when you download a release, you get the packaged libraries with that release. However, using `cargo`, there is no standard way of adding additional files to the binary release and package them. Therefore, this is a bit awkward at the moment.

Reply:
ohhh I see. Thanks for clearing that up. Maybe this should be noted somewhere else in the readme then? Because I am probably not the only one who installed it via cargo.

Reply:
I agree. This should probably be noted down somewhere (preferably the readme) and the error message should point to the readme when the libraries cannot be found,